contrib/mesos/docs/ha.md

```diff
@@ -0,0 +1,66 @@
+## High Availability
+
+### Scheduler
+
+Implementation of scheduler HA features has started and includes:
+
+- Checkpointing by default (`--checkpoint`)
+- Large failover-timeout by default (`--failover_timeout`)
+- Hot-failover w/ multiple scheduler instances (`--ha`)
+- Best effort task reconciliation on failover
+
+#### Multiple Instances
+
+Multiple scheduler instances may be run to support a hot-failover scenario in which one scheduler fails and another takes over immediately.
```

Comment:
Still says hot-standby. It's not a big deal but I think this is technically a warm standby (look at how the RedHat folks defined it here: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/proposals/high-availability.md)